### ISSUE TITLE
[change] generates a warning

### ISSUE THREAD

**Issue by chrissie-c** [enzienaudio/heavy#118]
Any use of the [change] object in a patch generates a warning during compilation:

"Warning: [route bang set] in "_main.pd/change.pd" @ (x:22, y:48): Heavy interprets route arguments such as "bang", "list", "float", and "symbol" literally. They cannot be used to filter generic messages as in Pd."

Which might or might nor be true, but seems a confusing thing to see when using what (on the face of it to this novice) seems a fairly simple operator :)

**Comment by diplojocus** [comment]
@chrissie-c yeah you're right, we should probably tone down the warnings here!
